package.json

```diff
@@ -0,0 +1,61 @@
+{
+  "name": "contact-list",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.jsx",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "build:dev": "webpack-dev-server --env dev",
+    "build:prod": "webpack-dev-server --env prod",
+    "lint": "eslint src/**/*.jsx"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/marishahv/ReactTask.git"
+  },
+  "author": "",
+  "license": "ISC",
+  "bugs": {
+    "url": "https://github.com/marishahv/ReactTask/issues"
+  },
+  "homepage": "https://github.com/marishahv/ReactTask#readme",
+  "dependencies": {
+    "bootstrap": "^4.3.1",
+    "npm": "^6.9.0",
+    "react": "^16.8.6",
+    "react-dom": "^16.8.6",
+    "react-redux": "^7.0.3",
+    "react-router": "^5.0.0",
+    "react-router-dom": "^5.0.0",
+    "redux": "^4.0.1",
+    "webpack-cli": "^3.3.2",
```

Comment:
All packages related to webpack should be dev dependecies

Reply:
Fixed